archive/zip_archiver.go

```diff
@@ -60,7 +60,7 @@ func (a *ZipArchiver) ArchiveFile(infilename string) error {
 	fh.Name = fi.Name()
 	fh.Method = zip.Deflate
 	// fh.Modified alone isn't enough when using a zero value
-	fh.SetModTime(time.Time{})
+	fh.SetModTime(time.Time{}) //nolint
```

Comment:
imo it'd be better if we fixed the linting issue here, even if that means splitting it out?

Reply:
The lint issue is to not call `SetModTime` but its required because its not fully deprecated.

Reply:
https://golang.org/pkg/archive/zip/#FileHeader.SetModTime

> Deprecated: Use Modified instead.

I am pretty sure per our testing though, you actually can't use `Modified` when you use a zero value